### ISSUE TITLE
Error when querying bugs with 1+ parameters

### ISSUE THREAD

**Issue by danisztls** [MichaelMure/git-bug#1032]
When running the following command from the README:

```shell
git bug ls "status:open sort:edit"
```

It returns `Error: unknown qualifier "status:open sort"`

Though this works fine:

```shell
git bug ls "status:open" "sort:edit"
```

**Comment by smoyer64** [comment]
It also works without any quotes at all - I'm a bit surprised that it didn't consider the whole enclosed string as a text query.  The next example in the README definitely requires the quote (in Bash on Linux) to escape the space:

```
git bug ls "foo bar" baz
```

Clearly it would be possible to fix the README but making the argument parsing more robust would be more user-friendly.  I'll take a look at whether the command-line parsing code in Cobra is platform dependent.  I'm also wondering if there's a way to test this for all OSes and shells we support.

**Comment by danisztls** [comment]
> I'm also wondering if there's a way to test this for all OSes and shells we support.

Can use a GH workflow for that and compare command outputs with expected values.

**Comment by smoyer64** [comment]
Yes ... there are a ton of tests running via GHA on Windows, Mac OSX and Linux in 'git-bug'.  What I meant is that we may not be able to test with every permutation of shell and OS, and that during testing (at least using `go test`), we can't control how the shell might process the arguments before it passes them to Go.
